src/cromshell/validate/command.py

```diff
@@ -22,12 +22,12 @@ class MissingArgumentError(Exception):
 @click.argument("wdl_json", type=click.Path(exists=True), required=False)
 @click.option(
     "-d",
-    "--dependencies",
+    "--dependencies-zip",
     required=False,
-    multiple=True,
     type=click.Path(exists=True),
-    help="MiniWDL option: Directory containing workflow source files that are "
-    "used to resolve local imports. (can supply multiple times)",
+    help="MiniWDL option: ZIP file or directory containing workflow source files "
```

Comment:
accessible to... what?  WHAT is it accessible to?  The mystery is killing me! :)